### PR TITLE
Check Sanity Tests for TrComp Images

### PR DESCRIPTION
**Pull request by shantanutrip**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
PT Trcomp 1.11 images seem to hang on the sanity tests on the Mainline context. Making this PR to verify how it behaves on the PR Context.

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: huggingface/pytorch/buildspec-trcomp.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ images:
   BuildHuggingFacePytorchTrCompGpuPy38Cu113TrainingDockerImage:
     <<: *TRAINING_REPOSITORY
     build: &HUGGINGFACE_PYTORCH_TRCOMP_GPU_TRAINING_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 18000
+    image_size_baseline: &IMAGE_SIZE_BASELINE 19000
```
Comment (shantanutrip) [inline huggingface/pytorch/buildspec-trcomp.yml:21]: Will revert - just changed to start the builds

File: test/dlc_tests/sanity/test_pre_release.py [hunk 2]
```diff
@@ -451,7 +451,7 @@ def _run_dependency_check_test(image, ec2_connection):
             "0.4": ["cpu", "gpu"],
             "0.5": ["cpu", "gpu"]
         },
-        "huggingface_pytorch_trcomp": {"1.9": ["gpu"]},
+        "huggingface_pytorch_trcomp": {"1.9": ["gpu"], "1.11": ["gpu"]},
```
Comment (shantanutrip) [inline test/dlc_tests/sanity/test_pre_release.py:454]: Currently installed version of openssl in the image is `openssl/now 1.1.1f-1ubuntu2.16 amd64` which is free from vulnerability.

**Review by harishneit — DISMISSED** [review]

**Review by harishneit — DISMISSED** [review]

**Review by harishneit — APPROVED** [review]

File: test/dlc_tests/sanity/test_telemetry.py [hunk 3]
```diff
@@ -136,7 +136,9 @@ def _run_instance_role_disabled(image_uri, ec2_client, ec2_instance, ec2_connect
     docker_cmd = "nvidia-docker" if processor == "gpu" else "docker"
 
     test_utils.login_to_ecr_registry(ec2_connection, account_id, image_region)
-    ec2_connection.run(f"{docker_cmd} pull -q {image_uri}")
+    ## For big images like trcomp, the ec2_connection.run command stops listening and the code hangs here.
+    ## Hence, avoiding the use of -q to let the connection remain active.
```
Comment (saimidu) [inline test/dlc_tests/sanity/test_telemetry.py:140]: Thanks for the explanatory comment here for future reference!
